derive_builder/src/options/struct_mode.rs

```diff
@@ -70,14 +70,23 @@ impl OptionsBuilderMode for StructMode {
 
 impl From<OptionsBuilder<StructMode>> for (StructOptions, OptionsBuilder<FieldMode>) {
     fn from(b: OptionsBuilder<StructMode>) -> (StructOptions, OptionsBuilder<FieldMode>) {
+        let field_default = match b.default_expression {
+            Some(DefaultExpression::Trait) 
```

Comment:
@colin-kiegel I think at this point you're right that DefaultExpression may be able to fall back to just having two variants: "Inherit" and "Explicit"